examples/visualization/plot_parallel_coordinate.ipynb

```diff
@@ -4,10 +4,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing High-dimentional Parameter Relationships in Jupyter Notebook\n",
+    "# Visualizing High-dimensional Parameter Relationships\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
-    "After optimizing the hyperparameter of neural networks, `plot_parallel_coordinate()` plots high-dimentional parameter relationships in a study.\n",
+    "After optimizing the hyperparameter of neural networks, `plot_parallel_coordinate()` plots high-dimensional parameter relationships in a study.\n",
```

Comment:
IMO, it is helpful to provide a link to the reference like `plot_intermediate_values.ipynb`.
```suggestion
    "After optimizing the hyperparameter of neural networks, [`plot_parallel_coordinate()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_parallel_coordinate) plots high-dimensional parameter relationships in a study.\n",
```